app/styles/new-signup.css

```diff
@@ -0,0 +1,88 @@
+.container {
+    width: 100%;
+    color: #666;
+    text-align: center;
+    padding: 1em 0;
+}
+
+.user-details {
+    display: flex;
+    padding: 2rem;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+    max-width: 80%;
+    margin: auto;
+    background-color: #f0f2f5;
+}
+
+.user-details>* {
+    padding: 1rem;
+}
+
+.user-details__heading {
+    font-size: 2rem;
+    color: black;
+    margin-bottom: 0;
+}
+
+.user-details__field input {
+    padding: 0.8rem;
+    width: 100%;
+    border-radius: 10px;
+    border: 2px solid darkgray;
+    font-size: 2rem;
+}
+
+.signup-action__btn button {
+    padding: .75rem 2rem;
+    border-radius: 10px;
+    outline: none;
+    background-color: green;
+    color: white;
+    text-decoration: none;
+    cursor: pointer;
+    width: 40%;
+    display: block;
+    margin: auto;
+    font-size: 2rem;
+    border: none;
+}
+.error {
+    color: red;
+    padding: 0.4rem;
+    font-size: 1rem;
+}
+.signup-action__btn button.btn-disabled {
```

Comment:
Can we please provide a one line space after each block of CSS ?

Reply:
okay I will do that